Tools/machines/frontier-olcf/install_dependencies.sh

```diff
@@ -97,6 +99,14 @@ python3 -m pip install --upgrade matplotlib
 python3 -m pip install --upgrade yt
 # install or update WarpX dependencies such as picmistandard
 python3 -m pip install --upgrade -r $HOME/src/warpx/requirements.txt
+# cupy for ROCm
+#   https://docs.cupy.dev/en/stable/install.html#building-cupy-for-rocm-from-source
+#   https://github.com/cupy/cupy/issues/7830
+CC=cc CXX=CC \
+CUPY_INSTALL_USE_HIP=1  \
+ROCM_HOME=${ROCM_PATH}  \
+HCC_AMDGPU_TARGET=${AMREX_AMD_ARCH}  \
+  python3 -m pip install cupy
```

Comment:
For some output instead of just a wheel `...` message
```suggestion
  python3 -m pip install -v cupy
```